src/visualizations/scatter-basic.js

```diff
@@ -0,0 +1,186 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import {
+  Scatter,
+  Axes,
+  // PieDonut,
+  Markers,
+  HotZone,
+} from '../parts'
+
+import { withParentSize } from '@vx/responsive'
+import { withTooltip, TooltipWithBounds } from '@vx/tooltip'
+
+import { numTicksForHeight, numTicksForWidth } from '../utils/responsive'
+import { scaleLinear, scaleBand } from '@vx/scale'
+import { extent } from 'd3-array'
+import { Group } from '@vx/group'
+// import { GlyphDot } from '@vx/glyph'
+import { localPoint } from '@vx/event'
+
+const ScatterBasic = ({
+  // withParentSize
+  parentWidth: width,
+  parentHeight: height,
+  // withToolTip
+  showTooltip,
+  hideTooltip,
+  tooltipOpen,
+  tooltipLeft,
+  tooltipTop,
+  tooltipData,
+  // own props
+  data,
+  margin,
+  yAxisLabel,
+  xAxisLabel,
+  drawAScatter,
+}) => {
+
+  const xMax = width - margin.left - margin.right
+  const yMax = height - margin.bottom - margin.top
+
+  const kGetter = (obj) => obj.fieldName
+  const vGetter = (obj) => obj.value
+
+  const xScale = scaleBand({
+    domain: data.map(kGetter),
+    range: [0, xMax],
+    clamp: true,
+  })
+
+  const yScale = scaleLinear({
+    domain: extent(data, vGetter),
+    range: [yMax, 0],
+    clamp: true,
+  })
+
+  const renderAxes = () => {
+    return (
+      <Axes
+        showGrid={false}
+        xMax={xMax}
+        yMax={yMax}
+        left={{
+          top: 0,
+          left: 0,
+          scale: yScale,
+          numTicks: numTicksForHeight(height),
+          label: yAxisLabel,
+        }}
+        bottom={{
+          top: yMax,
+          left: 0,
+          scale: xScale,
+          numTicks: numTicksForWidth(width),
+          label: xAxisLabel,
+        }}
+      />
+    )
+  }
+
+  // eslint-disable-next-line
```

Comment:
There's no need to use linter disable (and frankly we should minimize the use of it), since you're trying to handle the case of unused variable but the unused variable is a positional argument of the function handler -- simply remove it:

```js
const scatterShape = (d) => { }
```

will do